src/test/java/com/automationpracticetest/LogInPageTest.java

```diff
@@ -0,0 +1,65 @@
+package com.automationpracticetest;
+
+import com.automationpractice.LogInPage;
+import com.automationpractice.MyAccountPage;
+import com.utils.TestListener;
+import com.utils.WebDriverSingleton;
+import com.utils.BrowserNames;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+@Listeners(TestListener.class)
```

Comment:
In general, this is Listener implementation for making screenshots for failed tests. I specified it here for cases when it is needed to run separate tests from this class, otherwise, if to remove this line, Listener implementation will not work in this case which, I guess, is expected. But it still will work if to run tests using testng.xml file where this Listener is mentioned as well.

Reply:
Please try to use [Logging Listeners](https://testng.org/doc/documentation-main.html#logging-listeners)

Reply:
Thanks for the link. Tried implementation with using "TestListenerAdapter" and so far removing Listener-related references from everywhere (tests and testng) results in not taking screenshots. Let's discuss this case on Friday.